backend/ci/install_dependencies.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env sh
 
-pip-sync ./dev-requirements.txt ./requirements.txt
```

Comment:
What is the reason you changed this call?

Reply:
I wanted to make all command lines in `ci/` consistent for python tools by making them start with `python -m`. Other than that, it doesn't make any difference.